app/src/main/java/com/poupa/vinylmusicplayer/service/notification/PlayingNotificationImpl24.java

```diff
@@ -50,12 +51,12 @@ public synchronized void update() {
 
         Intent action = new Intent(service, MainActivity.class);
         action.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TOP);
-        final PendingIntent clickIntent = PendingIntent.getActivity(service, 0, action, 0);
+        final PendingIntent clickIntent = PendingIntentCompat.getActivity(service, 0, action, 0);
 
         final ComponentName serviceName = new ComponentName(service, MusicService.class);
         Intent intent = new Intent(MusicService.ACTION_QUIT);
         intent.setComponent(serviceName);
-        final PendingIntent deleteIntent = PendingIntent.getService(service, 0, intent, 0);
+        final PendingIntent deleteIntent = PendingIntentCompat.getService(service, 0, action, 0);
```

Comment:
Is the change on 3rd parameter on purpose? ie.  from `intent` to `action`

Reply:
Good catch

Reply:
Corrected